File: Php/app/Urls/RunningCashgameUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class RunningCashgameUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::runningCashgame, $homegame);
		}

	}

}

File: Web/Models/RunningCashgameUrlModel.cs
using Core.Classes;
using Web.Models;
using Web.Routing;

namespace app{

	public class RunningCashgameUrlModel : HomegameUrlModel{

		public RunningCashgameUrlModel(Homegame homegame) : base(RouteFormats.RunningCashgame, homegame){}

	}

}